scripts/run_tests.sh

```diff
@@ -57,8 +57,8 @@ done
 INDENT=0
 
 log() {
-	echo -n "[$(date)] "
-	head -c $(( INDENT * 4 )) /dev/zero | tr '\0' ' '
+	printf "[$(date)] "
```

Comment:
Shellcheck complains about thtis.  Can you use `printf "%s" "[$(date)] "` instead?